native/zowed/server.hpp

```diff
@@ -29,8 +29,9 @@ enum
   INVALID_REQUEST = -32600,  // The JSON sent is not a valid Request object
   METHOD_NOT_FOUND = -32601, // The method does not exist / is not available
   INVALID_PARAMS = -32602,   // Invalid method parameter(s)
-  INTERNAL_ERROR = -32603    // Internal JSON-RPC error
+  INTERNAL_ERROR = -32603,   // Internal JSON-RPC error
   // -32000 to -32099 are reserved for implementation-defined server-errors
+  REQUEST_TIMEOUT = -32001 // Request exceeded timeout limit
```

Comment:
Love this! 😻 

Dumb questions:
Where do we plan to document these? 
Or do we not expect users to ever see these RPC error codes?